pkg/image/stage.go

```diff
@@ -39,9 +39,19 @@ func (id StageID) IsEqual(another StageID) bool {
 	return (id.Digest == another.Digest) && (id.CreationTs == another.CreationTs)
 }
 
+type StageDescMeta struct {
```

Comment:
BaseImageSourceType    string
	BaseImageSourceAddress string

Not used in report